dev/make-distribution.sh

```diff
@@ -201,6 +201,12 @@ fi
 # Copy data files
 cp -r "$SPARK_HOME/data" "$DISTDIR"
 
+# Make pip package
+echo "Building python distribution package"
+cd $SPARK_HOME/python
+python setup.py sdist
```

Comment:
Almost everything else is built by default (except for maven profiles and the big tgz file). I don't think adding a parameter would really improve it (although if on some machines people have difficulty or its slow we can revist - but so far building the Python distribution is very fast).